packages/webapp/src/components/Finances/FinancesCarrousel/index.jsx

```diff
@@ -0,0 +1,222 @@
+import { ReactComponent as CropIcon } from '../../../assets/images/finance/Crop-icn.svg';
+import styles from './styles.module.scss';
+import { Semibold, Text } from '../../Typography';
+import clsx from 'clsx';
+import TextButton from '../../Form/Button/TextButton';
+
+ChartJS.register(ArcElement, Tooltip);
+
+const ShadowPlugin = {
+  beforeDraw: (chart) => {
+    const { ctx } = chart;
+    ctx.shadowColor = '#FFEFCF';
+    ctx.shadowBlur = 0;
+    ctx.shadowOffsetX = 0;
+    ctx.shadowOffsetY = 4;
+  },
+};
+
+const FinancesCarrousel = ({
+  totalExpense,
+  totalRevenue,
+  labourExpense,
+  otherExpense,
+  estimatedRevenue,
+  currencySymbol,
+  history,
+}) => {
+  const { t } = useTranslation();
+
+  const expenseChartData = {
+    labels: ['Total other', 'Total labour'],
+    datasets: [
+      {
+        data: [11500, 21523],
+        backgroundColor: ['#FFB800', '#AA5F04'],
+      },
+    ],
+  };
+
+  const profitLoss = totalRevenue - totalExpense;
+
+  const cards = [
+    {
+      id: 'profit-loss',
+      label: t('SALE.FINANCES.PROFIT_LOSS'),
+      inactiveBackgroundColor: 'var(--teal700)',
+      inactiveIcon: <ProfitLossIconDark role="img" aria-label={t('SALE.FINANCES.PROFIT_LOSS')} />,
+      activeContent: (
+        <div className={clsx([styles.cardContent, styles.profitLossCardContent])}>
+          <div className={styles.revenueExpensesContainer}>
+            <TextButton
+              className={styles.revenueContainer}
+              onClick={() => history.push('/finances/actual_revenue')}
+            >
+              <div>
+                <Text className={styles.revenueTitle}>{t('SALE.FINANCES.TOTAL_REVENUE')}</Text>
+                <p className={clsx([styles.stat, styles.revenueStat])}>
+                  {currencySymbol}
+                  {totalRevenue}
+                </p>
+              </div>
+              <BsChevronRight />
+            </TextButton>
+            <div className={styles.expenseContainer}>
+              <Text className={styles.expenseTitle}>{t('SALE.FINANCES.TOTAL_EXPENSES')}</Text>
+              <p className={clsx([styles.stat, styles.expenseStat])}>
+                {currencySymbol}
+                {totalExpense}
+              </p>
+            </div>
+          </div>
+          <div className={styles.profitLossSummaryContainer}>
+            <ProfitLossIconLight role="img" aria-label={t('SALE.FINANCES.PROFIT_LOSS')} />
+            <Text className={styles.profitLossTitle}>{t('SALE.FINANCES.PROFIT_LOSS')}</Text>
+            <p className={clsx([styles.stat, styles.profitLossStat])}>
+              {profitLoss < 0 ? '-' : ''}
+              {currencySymbol}
+              {Math.abs(profitLoss)}
+            </p>
+          </div>
+        </div>
+      ),
+      note: t('SALE.FINANCES.CARROUSEL_TEXT.GENERIC'),
+      noteColor: '#72B8B0',
+    },
+    {
+      id: 'expenses',
+      label: t('SALE.FINANCES.EXPENSES'),
+      inactiveBackgroundColor: 'var(--white)',
```

Comment:
We haven't defined the value `white`. I think we can add `#ffffff`, but we've been doing `var(--white, #fff)` 😅